src/neuralnet/accrual/snapshot.h

```diff
@@ -195,7 +195,7 @@ class SnapshotCalculator
     //! \brief Calculate the age of the active superblock to determine the
     //! duration of the accrual period.
     //!
-    //! \return Superblock age as days in until to the payment time.
+    //! \return Superblock age as seconds in until to the payment time.
```

Comment:
```suggestion
    //! \return Superblock age as seconds from the payment time.
```

Wow, not sure what I was originally trying to blurt out there...

Reply:
Oh wow don't know how I missed that. Squashed that into the original commit and changed the title to match